iphone/Classes/TiApp.m

```diff
@@ -799,17 +799,19 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didSend
 
 -(void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)dataTask didReceiveData:(NSData *)data
 {
-    if (!uploadTaskResponses) {
-        uploadTaskResponses = [[[NSMutableDictionary alloc] init] retain];
+    if(!nsURLUploadTaskResponses){
+        nsURLUploadTaskResponses = [[NSMutableDictionary alloc] init];
```

Comment:
This changes overwrote the old changes we did (formatting, variable naming, etc.). Please restore that, I really want to include it soon..